.env

```diff
@@ -0,0 +1,2 @@
+PORT=8002
```

Comment:
This port should be unique, and should be reserved in the [ports table](https://github.com/canonical-web-and-design/practices/blob/master/project-structure/ports.md)